CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_stripe_source v0.9.3
+PR [#](https://github.com/fivetran/dbt_stripe_source/pull/) includes the following updates:
+## Under the Hood:
+- Move the live_mode predicate to the non *_tmp staging models. This is because the [0.4.3 release of fivetran_utils](https://github.com/fivetran/dbt_fivetran_utils/releases/tag/v0.4.3) infers if the source table a package is present, and if not the package will create an empty table to ensure the downstream transformations succeed. Previously the Stripe package wasn't compatible because of where the live_mode predicate existed. In the case there will be an empty data model created, we needed to move the predicate so that the queries and downstream models will still run correctly.
```

Comment:
Minor suggestion to rephrase the CHANGELOG. We can highlight the main change in a single bullet, then add a deeper description as a sub bullet.

```suggestion
- The live_mode predicate macro has been moved to the non *_tmp staging models. 
     - This move is necessary due to the [0.4.3 release of fivetran_utils](https://github.com/fivetran/dbt_fivetran_utils/releases/tag/v0.4.3) infers if the source table a package is present, and if not the package will create an empty table to ensure the downstream transformations succeed. Previously the Stripe package wasn't compatible because of where the live_mode predicate existed. In the case there will be an empty data model created, we needed to move the predicate so that the queries and downstream models will still run correctly.
```

Reply:
gotcha